src/features/ITGovernance/Admin/GRBReview/GRBReviewForm/ReviewType/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Controller } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { Fieldset, FormGroup } from '@trussworks/react-uswds';
+import {
+  SystemIntakeGRBReviewFragment,
+  SystemIntakeGRBReviewType,
+  useUpdateSystemIntakeGRBReviewTypeMutation
+} from 'gql/generated/graphql';
+
+import { EasiFormProvider, useEasiForm } from 'components/EasiForm';
+import { RadioField } from 'components/RadioField';
+import RequiredAsterisk from 'components/RequiredAsterisk';
+
+import GRBReviewFormStepWrapper, {
+  GRBReviewFormStepSubmit
+} from '../GRBReviewFormStepWrapper';
+
+type ReviewTypeFields = {
+  grbReviewType: SystemIntakeGRBReviewType;
+};
+
+type ReviewTypeProps = {
+  grbReview: SystemIntakeGRBReviewFragment;
+};
+
+const ReviewType = ({ grbReview }: ReviewTypeProps) => {
+  const { t } = useTranslation('grbReview');
+
+  const [mutate] = useUpdateSystemIntakeGRBReviewTypeMutation();
+
+  const form = useEasiForm<ReviewTypeFields>({
+    defaultValues: {
+      grbReviewType: grbReview.grbReviewType
+    }
+  });
+
+  const {
+    control,
+    formState: { errors }
+  } = form;
+
+  const onSubmit: GRBReviewFormStepSubmit<ReviewTypeFields> = async input =>
+    mutate({ variables: { input } });
+
+  return (
+    <EasiFormProvider<ReviewTypeFields> {...form}>
+      <GRBReviewFormStepWrapper<ReviewTypeFields>
+        onSubmit={onSubmit}
+        grbReview={grbReview}
+      >
+        <FormGroup error={!!errors.grbReviewType} className="margin-top-5">
+          <Fieldset>
+            <legend className="text-bold">
+              {t('setUpGrbReviewForm.reviewType.label')} <RequiredAsterisk />
+            </legend>
```

Comment:
Hi 👋🏼  was just looking around the code while it was still in draft mode. 
Truss has a built in "required marker" to their `Label` component. 
To use: `<Label requiredMarker />`

Label props below:
```
declare type LabelProps = {
    children: React.ReactNode;
    htmlFor: string;
    className?: string;
    error?: boolean;
    hint?: React.ReactNode;
    srOnly?: boolean;
    requiredMarker?: boolean;
};
```

Reply:
Thanks for pointing that out! Looks like that prop wasn't available before we upgraded `react-uswds`, so I hadn't noticed the change. This is a `legend` instead of `label`, but I see that the `Fieldset` component also added a `requiredMarker` prop that gets added to the `legend`.

I usually just use a separate `legend` element because there isn't a way to make the `Fieldset` component's legend bold without either adding CSS or wrapping it in a `span` tag within the `legend` prop.